### PR TITLE
feat(create-app): add vue3 typescript static template

### PR DESCRIPTION
**Pull request by keyding**
### Relates Issue

#348 

### Main changes

Adding Vue3 TypeScript Static template project.

### Screenshots

Choose template:
choose-template

Generate Project:
generate-project

### Check list

- [x] Project create by create-blocklet can work with `npm run dev`
- [x] Project create by create-blocklet can work with `npm run deploy`
- [x] Project create by create-blocklet can work with `npm run upload`
- [x] If a new version need to be publish, make sure you have run `npm run bump-version` command

### CODE REVIEW

File: packages/create-app/templates/vue-ts-static/blocklet.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: template-vue-ts
+title: Blocklet Template Vue + TypeScript
+description: A Blocklet Server blocklet
+keywords:
+  - blocklet
+  - vue
+  - typescript
+group: static
+did: z8iZh3SGuBsFsBW9TFk6paXCQUfDaapfJAYjN
```
Comment (LancelotLewis) [inline packages/create-app/templates/vue-ts-static/blocklet.yml:9]: did field in template should be empty

**Review by LancelotLewis — CHANGES_REQUESTED** [review]

File: packages/create-app/templates/vue-ts-static/blocklet.yml (hunk 1 above)
Comment (keyding) [reply packages/create-app/templates/vue-ts-static/blocklet.yml]: OK. The did field has been cleared.

**Review by LancelotLewis — APPROVED** [review]
LGTM

**Comment by LancelotLewis** [comment]
@keyding
you need to run `npm run bump-version` in this project folder

**Comment by keyding** [comment]
> @keyding you need to run `npm run bump-version` in this project folder

Yes, I'm done.

**Comment by LancelotLewis** [comment]
@keyding Please bump-version again 🤣

**Comment by keyding** [comment]
> @keyding Please bump-version again 🤣

Done.

**Pull request merged**